docs/source/troubleshooting/logs.rst

```diff
@@ -0,0 +1,153 @@
+==================================
+Logs on NI Linux Real-Time Systems
+==================================
+
+Introduction
+============
+
+This document is an introduction to some of the common logs on Linux Real-Time Systems.
+
+Useful NI Linux Real-Time Logs
+==============================
+
+Logs in */var/log/*
+-------------------
+
+The Linux Operating System and many Linux utilities will write to ``/var/log/`` by default.
+For example, the general ``messages`` log (or``syslog`` depending on the system) is typically found here.
+
+NI Linux Real-Time Systems are designed as embedded, headless systems that need to be able to recover should issues occur.
+As such, the ``/var/log/`` directory is a symbolic link to ``/var/volatile/log/``.
+The ``/var/volatile/log/`` directory is mapped to RAM instead of the persistent device storage.
+This prevents the system’s device storage from filling with logs and causing other issues on devices with extremely limited storage but results in the logs not persisting through a reboot.
+To make these logs persist through a reboot, see the Remapping Volatile Logs section of this document.
+
+**Note:** The ``kern.log`` file is traditionally stored here, but on NI Linux Real-Time systems is instead located at ``/var/local/natinst/log/`` to ensure it persists between reboots.
+
+.. csv-table:: NI Linux RT Logs
+   :file: resources/nilrt-logs.csv
+   :widths: 30, 30, 50
+   :header-rows: 1
+
+Logs in */var/local/natinst/log/*
+---------------------------------
+
+Most NI Software will log to the ``/var/local/natinst/log/`` directory.
+The following logs will exist by default with LabVIEW Real-Time installed.
```

Comment:
I expected a list of logs here, is there a list somewhere? 

I would of course be fine with no list, since we may change the list over time...

Reply:
The list is actually in the ni-logs.csv file in this PR. The `.. csv-table::` block below uses the values from the csv to create the table.